src/Action.tsx

```diff
@@ -8,25 +8,33 @@ interface SelectFileProps
   prompt?: string;
   type?: string;
   onSelect(filePath: string): unknown;
+  onError(): unknown;
```

Comment:
Do we need this? Wouldn't it be enough to call `onSelect` with `null` or `undefined` maybe?

Reply:
naming wise `onSelect` feels a file is selected, hence `onError`, should I return `null` in onSelect then?

Reply:
It's not really an error either, it's just that the user did select nothing?